module/mfrc522_user_command.c

```diff
@@ -115,7 +119,7 @@ static int generate_random(char *answer)
 	 * but at the next mem_read command. So we don't check mem_read return
 	 * value
 	 */
-	mem_read(buffer);
+	mem_read(buffer, stats);
```

Comment:
This `mem_read` is only here for debugging purposes and will invalidate the results. The user shouldn't realize that we read that memory here, since it's only to display the random ID before we use `cat` or `hexdump` on our device. We should be able to pass a NULL stats pointer in order to allow not aggregating statistics.

Reply:
This is still present in the last commits @n1tram1 

```suggestion
	mem_read(buffer, NULL);
```